core/src/main/scala/dagr/core/cmdline/parsing/DagrCommandLineParser.scala

```diff
@@ -25,53 +25,21 @@ package dagr.core.cmdline.parsing
 
 import java.nio.file.Files
 
+import dagr.commons.io.PathUtil
 import dagr.core.cmdline._
-import dagr.core.cmdline.parsing.CommandLineParserStrings.UsagePrefix
-import dagr.core.cmdline.parsing.ParsingUtil._
-import dagr.core.config.Configuration
-import dagr.core.tasksystem.{Pipeline, ValidationException}
-import dagr.core.util.StringUtil._
-import dagr.core.util.{PathUtil, StringUtil}
-import DagrCommandLineParserStrings._
-import ParsingUtil._
-
-import scala.collection.JavaConversions._
-import scala.collection.mutable.ListBuffer
-import scala.collection.{Map, Set, mutable}
+import dagr.core.tasksystem.Pipeline
+import dagr.sopt.cmdline._
+import dagr.sopt.util.ParsingUtil._
 
 /** Stores strings used in the usage and error messages */
-private[parsing] object DagrCommandLineParserStrings {
-  val AvailablePipelines = "Available Pipelines:"
-  val MissingPipelineName = "No pipeline name given."
-  val SeparatorLine = wrapString(KWHT, s"--------------------------------------------------------------------------------------\n", KNRM)
-  val PipelineGroupNameColumnLength = 48
-  val PipelineGroupDescriptionColumnLength = 45
-  val PipelineNameColumnLength = 45
-
+object DagrCommandLineParserStrings {
   def unknownPipelineErrorMessage(commandLineName: String): String = {
     s"No known pipeline name found.  If using --scripts, try using '--' separator between $commandLineName options and Pipeline name."
   }
-
-  def unknownCommandErrorMessage(command: String): String = {
-    s"'$command' is not a valid command. See DagrCommandLine --help for more information."
-  }
-}
-
-private[cmdline] object DagrCommandLineParser {
-  /** The maximum line lengths for pipeline descriptions */
-  val MaximumLineLength = 80
-
-  /** Similarity floor for matching in printUnknown **/
-  val HelpSimilarityFloor: Int = 7
-  val MinimumSubstringLength: Int = 5
 }
 
 /** Methods for parsing among top level command line tasks */
 private[cmdline] class DagrCommandLineParser(val commandLineName: String, val includeHidden: Boolean = false) {
```

Comment:
Now that this is _so_ much simpler, and it's only real use is from DagrMain, I think it would be really nice to merge those two classes.